README.md

```diff
@@ -197,9 +236,9 @@ A style guide for the [HTML Template Language](https://docs.adobe.com/docs/en/ht
     </sly>
     ```
     
-  - [4.4](#4.4) <a name='4.4'></a> **Use camelcase for identifier names.**
+  - [4.4](#4.4) <a name='4.4'></a> **Use lowerCamelCase for identifier names.**
   
-    Using camelCase will help to increase the readability of your identifiers. Notice though that
+    Using lowerCamelCase will help to increase the readability of your identifiers. Notice though that
```

Comment:
I would add a link to an explanation of what "lowerCamelCase" is (or an example).

Reply:
Good, idea, I'll add a definition.

Reply:
Done!